spec/rails_translation_manager/locale_checker_spec.rb

```diff
@@ -3,7 +3,17 @@
 require "spec_helper"
 
 RSpec.describe RailsTranslationManager::LocaleChecker do
+  before do
+    allow($stdout).to receive(:puts)
+  end
```

Comment:
I agree. I still think  `allow($stdout).to receive(:puts)` is needed for certain tests where we are validating the return value, rather than the output. I attempted (unsuccessfully) to chain `eq` with the stdout handlers. Maybe there is another RSpec way to handle both output and return value?

Reply:
Do you have an example of one? My general view is `allow($stdout).to receive(:puts)` is a smell that's best resolved a different way. The RSpec way isn't amazing but still a lot cleaner.

Reply:
Oh I should have read further. I'll have a look at the code changed and get back to you.

Reply:
I've added a commit that consistently applies the `expect { }.to output.to_stdout` - feel free to remove it if you don't like it. I was able to also clear up a bunch of other writes to stdout with it.

It did come out a bit ugly in this class, but I think it's worth it for consistently and not resorting to stubbing $stdout. I do think this class could be improved it seems sub optimal for it to be both a boolean output and a write to stdout, they seem like they should be two separate methods - which would be easier to test.

P.s. excuse the late time here, I've adjusted my work hours today for an afternoon appointment.

Reply:
Thanks for standardising, happy with your changes (although 'ugly' in the locale checker spec 😄)! However, you make a good point about being two separate methods, which would lead to a cleaner solution / specs. As this work is drifting from the original intent of the PR, I'll leave it for now and think on improving the class in a future PR.

Reply:
Sounds good 👍